tests/ptf_runner.py

```diff
@@ -1,27 +1,37 @@
 import pipes
 
-def ptf_runner(host, testdir, testname, platform_dir, params={}, \
-               platform="remote", qlen=0, relax=True, debug_level="info", \
-               socket_recv_size=None, log_file=None):
+def ptf_runner(host, testdir, testname, platform_dir=None, params={},
+               platform="remote", qlen=0, relax=True, debug_level="info",
+               socket_recv_size=None, log_file=None, device_sockets=[]):
```

Comment:
It's a separate command line option in PTF. I think it was missed because the COPP tests are the only one that's using that command line option (as far as I can tell).

It might be worth seeing if it can be refactored into params going forward, but I think I'll save for that for a later PR.